p2p_wallet/Scenes/CreateOrRestoreWallet/RestoreWallet/RestoreICloud/RestoreICloud.ViewController.swift

```diff
@@ -49,23 +46,32 @@ extension RestoreICloud {
         // MARK: - Methods
         override func setUp() {
             super.setUp()
-            view.addSubview(headerView)
-            headerView.autoPinEdgesToSuperviewEdges(with: .init(all: 20), excludingEdge: .bottom)
+            
+            view.addSubview(navigationBar)
+            navigationBar.titleLabel.text = L10n.createANewWallet
+            navigationBar.autoPinEdge(toSuperviewSafeArea: .top)
+            navigationBar.autoPinEdge(toSuperviewEdge: .leading)
+            navigationBar.autoPinEdge(toSuperviewEdge: .trailing)
             
             view.addSubview(walletsCollectionView)
-            walletsCollectionView.autoPinEdge(.top, to: .bottom, of: headerView, withOffset: 30)
+            walletsCollectionView.autoPinEdge(.top, to: .bottom, of: navigationBar, withOffset: 0)
             walletsCollectionView.autoPinEdge(toSuperviewSafeArea: .leading)
             walletsCollectionView.autoPinEdge(toSuperviewSafeArea: .trailing)
             walletsCollectionView.autoPinEdge(toSuperviewEdge: .bottom, withInset: 30)
```

Comment:
The last row has not to be cut at the end like this:
![Simulator Screen Shot - iPhone 8 - 2021-11-16 at 11 25 33](https://user-images.githubusercontent.com/6975538/141917924-d1a84993-c5f9-4978-94dd-d9a7d591d59e.png)
Solution: remove `withInset: 30` (set offset to 0) and modify `contentInset` of collectionview
`walletsCollectionView.collectionView.modify(dBottom: 20)`